docs/madr/decisions/078-special-mtp-algo.md

```diff
@@ -0,0 +1,119 @@
+# MeshTrafficPermission Is a Special Case of Inbound Policies and Requires a Custom Matching Algorithm
+
+* Status: accepted
+
+## Context and Problem Statement
+
+In the [Inbound Policies MADR](https://docs.google.com/document/d/1tdIOVVYObHbGKX1AFhbPH3ZQKYvpQG-2s6ShHSECaXM) we said 
```

Comment:
How can this is not a MADR in markdown (or at least an number with a link to the google docs).

There's no point in having MADRs if there's no centralised source of truth.

Reply:
There is MADR with link https://github.com/kumahq/kuma/blob/abaf5c54ff421a82c447eb8e28a3561dbfaf0e38/docs/madr/decisions/066-inbound-policies.md#L7-L6, I just wanted to save few clicks for readers, but I can replace the link

Reply:
updated the link